### DISCUSSION TITLE
A500 WHDLoad games

### DISCUSSION THREAD

**Discussion by jazz1979** [BlitterStudio/amiberry#1444] [Ideas]
Hi, wrt Amiga emulation using retroarch and PUAE2021, it would be good to mention that for A500 games it is best to use A600 as the HD loading system in combinatoin with kick37350.A600 or kick40063.A600 for games to run smooth without stuttering, the A1200 is set as preferred system so the A600 should be selected in the quickmenu upon launching a game. i struggled with this for a while until someone helped me configure this. BR/Jazz

**Comment by giantclambake** [comment]
I don't understand ~ for non-AGA WHDLoad titles, amiberry uses  the A600HD machine type with KS ROM v2.05 rev 37.350 (512k)

**Comment by midwan** [comment]
PUAE is a different project, not related to this one. I'm not sure what you're trying to say either. Amiberry uses a mechanism to change the configuration automatically, depending on the title selected, if it's a recognized WHDLoad pack (`.lha`). It will switch from A600HD to A1200 (or CD32/CDTV), depending on the selected game/demo.

That does not happen with floppy images of course, but usually distros like RetroPie will check the filename and pick either an A500 or an A1200 model for you (depending on if AGA is mentioned in the filename, usually).

**Comment by jazz1979** [comment]
Ok, thanks, my bad, postred in the wrong place,  ignore my request  :)
